### DISCUSSION TITLE
Can't bypass amazon

### DISCUSSION THREAD

**Discussion by Elektronikgirl** [SmoothWAN/SmoothWAN#145] [General & Help]
Hello! 
I can find the bypass setting for my smoothWAN but amazon is grayed out. How can i turn on the bypass feature for amazon?

Thanks,

**Comment by Elektronikgirl** [comment]
> Hello! I can find the bypass setting for my smoothWAN but amazon is grayed out. How can i turn on the bypass feature for amazon?
> 
> Thanks,

And of course I go back to my window and it allowed it... I'm not sure what happened!

**Reply by TalalMash** [reply]
Speedify UI window? Does it happen with the fullscreen link?

**Reply by Elektronikgirl** [reply]
Yeah it was in the speedify UI window - I am pretty new to this so I don't know about the fullscreen link.
